src/main/java/net/sf/jabref/model/entry/FieldName.java

```diff
@@ -42,16 +43,35 @@
     public static final String NUMBER = "number";
     public static final String ORGANIZATION = "organization";
     public static final String PAGES = "pages";
+    public static final String PAGETOTAL = "pagetotal";
     public static final String PDF = "pdf";
     public static final String PS = "ps";
     public static final String PUBLISHER = "publisher";
+    public static final String REPORTNO = "reportno";
     public static final String REVIEW = "review";
     public static final String SCHOOL = "school";
     public static final String SERIES = "series";
+    public static final String SUBTITLE = "subtitle";
     public static final String TITLE = "title";
     public static final String TYPE = "type";
     public static final String URL = "url";
     public static final String URLDATE = "urldate";
+    public static final String VERSION = "version";
     public static final String VOLUME = "volume";
     public static final String YEAR = "year";
+    public static final String YEARFILED = "yearfiled";
+
+    // JabRef internal field names
+    public static final String OWNER = "owner";
+    public static final String TIMESTAMP = "timestamp"; // Not the actual field name, but the default value
+    public static final String NUMBER_COL = "#";
```

Comment:
Okay it is... https://github.com/JabRef/jabref/pull/1627/files#diff-59eb0b0c47b8c7b57710bde78ec9f349L215